relayer/scripts/run-relayer.sh

```diff
@@ -15,6 +15,10 @@ CONFIG_FILE=$5
 hermes --config $CONFIG_FILE keys add --chain $CHAIN1_ID --mnemonic-file $ALICE_KEY
 hermes --config $CONFIG_FILE keys add --chain $CHAIN2_ID --mnemonic-file $BOB_KEY
 
+echo "Waiting for the local chain to start"
+sleep 150
+echo "Local chain started"
```

Comment:
This seems incorrect and brittle. Message without verifying if the chain is started or not.

Reply:
Made an update at this [commit](https://github.com/agoric-labs/agoric-ibc-relayer/pull/1/commits/77e756e35702a8c065ff4b7f219f3338847aad38) 

It is very similar to how we used to do it in offer-up [over here](https://github.com/Agoric/dapp-offer-up/blob/e8a06006d1b870e2e0177f5bde110c9b37dfa987/.github/workflows/pr.yml#L39-L57C12)

Reply:
Made another update [over here](https://github.com/agoric-labs/agoric-ibc-relayer/pull/1/commits/6a1d25ed5517cb5f8b34aa4502b93afc98a24501)